modules/@apostrophecms/page/index.js

```diff
@@ -696,6 +696,8 @@ database.`);
         browserOptions.name = self.__meta.name;
         browserOptions.canPublish = self.apos.permission.can(req, 'publish', '@apostrophecms/any-page-type');
         browserOptions.quickCreate = self.options.quickCreate && self.apos.permission.can(req, 'edit', '@apostrophecms/any-page-type', 'draft');
+        browserOptions.localized = true;
```

Comment:
So that the front end can easily detect these facts about the page module like it can about the piece modules.